src/tests/mod.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg(feature = "kimchi")]
 mod examples;
```

Comment:
feels weird to gate this under kimchi, but I guess maybe less painful than add the feature gate within examples?